lib/fresh_service/config.ex

```diff
@@ -0,0 +1,50 @@
+defmodule FreshService.Config do
+  @moduledoc """
+  Stores configuration variables used to communicate with FreshService's API.
+
+  All settings also accept `{:system, "ENV_VAR_NAME"}` to read their
+  values from environment variables at runtime.
+  """
+
+  @doc """
+  Returns the FreshService Account username. Set it in `mix.exs`:
+
+      config :fresh_service, username: "YOUR_USERNAME"
+  """
+  def username, do: from_env(:username)
+
+  @doc """
+  Returns the FreshService Account password. Set it in `mix.exs`:
+
+      config :fresh_service, password: "YOUR_PASSWORD"
+  """
+  def password, do: from_env(:password)
+
+  @doc """
+  Returns the FreshService Api key for your account. Set it in `mix.exs`:
+
+      config :fresh_service, api_key: "YOUR_API_KEY"
+  """
+  def api_key, do: from_env(:api_key)
+
+  @doc """
+  Returns the domain of the FreshService. Set it in `mix.exs`:
+
+      config :fresh_service, domain: "YOUR_DOMAIN"
+  """
+  def domain, do: from_env(:domain)
```

Comment:
@kamil89 Why have you started implementing this abstraction? Can't we just read it from mix config instead of creating `from_env` function ?